library/javatests/net/openid/appauth/AuthorizationResponseTest.java

```diff
@@ -133,5 +136,7 @@ private void checkExpectedFields(AuthorizationResponse authResponse) {
                 TEST_ID_TOKEN, authResponse.idToken);
         assertEquals("access token expiration time does not match",
                 TEST_TOKEN_EXPIRE_TIME, authResponse.accessTokenExpirationTime);
+        assertEquals("scope does not match",
+                     TEST_SCOPE, authResponse.scope);
```

Comment:
You are testing the wrong class - your changes affect `TokenResponse`, not `AuthorizationResponse`. Please revert changes to `AuthorizationResponseTest` and add a test to `TokenResponseTest`.